enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/backup/BackupCoreIT.java

```diff
@@ -102,7 +102,12 @@ static String backupAddress( GraphDatabaseFacade db )
         InetSocketAddress inetSocketAddress = db.getDependencyResolver()
                 .resolveDependency( Config.class ).get( CausalClusteringSettings.transaction_advertised_address )
                 .socketAddress();
-        return inetSocketAddress.getHostName() + ":" + (inetSocketAddress.getPort() + 2000);
+
+        HostnamePort hostnamePort = db.getDependencyResolver()
+                .resolveDependency( Config.class )
+                .get( OnlineBackupSettings.online_backup_server );
+
+        return inetSocketAddress.getHostName() + ":" + hostnamePort.getPort();
```

Comment:
The old way was to take the Raft address and add 2000 to it. Now the backup port is set explicitly by test setup, and read here